submissions/Neskazhuk/popup/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Popup</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600&display=swap" rel="stylesheet">
+    <link href="styles.css" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <nav class="nav">
+            <ul class="nav-list">
+                <li class="nav-item"><a class="link" href="#" tabindex="1">Hello</a></li>
+                <li class="nav-item"><a class="link" href="#" tabindex="2">World</a></li>
+                <li class="nav-item popup">
+                    <input id="popup-input-id" class="popup-input" type="checkbox" tabindex="3">
```

Comment:
Same story about `tabindex` attribute on interactive elements

Reply:
I used it in order to complete the task: all interactive elements (including inputs) must be focusable without a mouse. Otherwise, a reader would omit the 'more' button.